geolocation/src/definitions.ts

```diff
@@ -0,0 +1,152 @@
+export type CallbackID = string;
+
+export type GeolocationState = PermissionState;
```

Comment:
This should be imported from `@capacitor/core`, see https://github.com/ionic-team/capacitor-plugins/pull/33#discussion_r543776894